static/css/style.css

```diff
@@ -1062,3 +1073,31 @@ code {
 .plugin-manual {
   font-size: 50px;
 }
+
+.container-search {
+  padding-top: 10px;
+}
+
+.blog-item {
+  padding-bottom: 30px;
+}
+
+.blog-item:not(:first-child) {
+  border-top: 1px #eee solid;
+}
+
+.metadata {
+  color: #777;
+}
+
+.post-content {
+  padding-top: 20px;
```

Comment:
I'd suggest removing this, there's a large gap between the blog post title and the content.